packages/app-mobile/services/e2ee/Krypto.react-native.ts

```diff
@@ -0,0 +1,51 @@
+import { KryptoInterface } from '@joplin/lib/services/e2ee/types';
+import crypto from 'react-native-quick-crypto';
+import { HashAlgorithm } from 'react-native-quick-crypto/lib/typescript/keys';
+
+const Krypto: KryptoInterface = {
+
+	getCiphers: (): string[] => {
+		return crypto.getCiphers();
+	},
+
+	getHashes: (): string[] => {
+		return crypto.getHashes();
+	},
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- TODO: remove the type "any" here
```

Comment:
You mentioned that the pull request is ready to be merged but surely we can't merge this. You're correct that all "any" should be replaced by proper types.

Reply:
I wonder if it's good to add `react-native-quick-crypto`(or `craftzdog/react-native-buffer`) in the dependencies of the whole project, so I can import the `Buffer` type in it as `RNBuffer` and make a new type `NodeBuffer | RNBuffer` for the interface.

Reply:
> and make a new type NodeBuffer | RNBuffer for the interface.

Ideally the interface would not mention any RN-specific type because it's going to be shared with non-RN code. Don't NodeBuffer and RNBuffer share the same interface? Are there differences? If it's the same, maybe create a new interface that declares just the methods and properties that you need

Reply:
I've created `CryptoBuffer` for it.